src/components/formik-forms/formik-radio-button.jsx

```diff
@@ -0,0 +1,106 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+import {Field} from 'formik';
+
+const FormikInput = require('./formik-input.jsx');
+
+require('./formik-radio-button.scss');
+require('../forms/row.scss');
+
+const FormikRadioButtonSubComponent = ({
```

Comment:
Question for my own understanding of React/Formik: How does this get props?  I see it passed as component=FormikRadioButtonSubComponent on the Field below. Is the formik field doing that work, or is is some other part of the React component system?

Reply:
Yes, the `Formik` component passes a bunch of props to the custom component that you tell it to use for radio buttons. 

From the docs, https://jaredpalmer.com/formik/docs/api/field:

```
Custom React Components will be passed FieldProps which is same render prop parameters of <Field render> plus any other props passed to directly to <Field>.
```